### PR TITLE
Fix infinite loop

### PR DESCRIPTION
**Pull request by MathuraMG**
Before your pull request is reviewed and merged, please ensure that:

* [ ] there are no linting errors
* [ ] your code is in a uniquely-named feature branch and has been rebased on top of the latest master. If you're asked to make more changes, make sure you rebase onto master then too!
* [ ] your pull request is descriptively named and links to an issue number, i.e. `Fixes #123`
* [ ] **Make sure you have run the tests!!**
  * [ ] Integration test & Server unit tests - run `npm run allTest` from the server folder
  * [ ] Client unit test - run `npm run test` from client folder
  * [ ] End to End test - run `npm run test` from the root folder

Thank you!

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -17,12 +17,14 @@
   "dependencies": {
     "axios": "^0.18.1",
     "babel-polyfill": "^6.26.0",
+    "babel-standalone": "^6.26.0",
```
Comment (MathuraMG) [inline client/package.json:20]: do we need this now?

File: client/src/app/components/App/Shared/EditorComponents/CodeOutput/FrontEndOutput.jsx [hunk 2]
```diff
@@ -12,6 +13,9 @@ class FrontEndOutput extends React.Component {
     super(props);
     this.startSketch = this.startSketch.bind(this);
     this.stopSketch = this.stopSketch.bind(this);
+    this.errorCallback = (line) => {
```
Comment (MathuraMG) [inline client/src/app/components/App/Shared/EditorComponents/CodeOutput/FrontEndOutput.jsx:16]: can we remove this?

File: client/src/app/components/App/Shared/EditorComponents/CodeOutput/FrontEndOutput.jsx [hunk 3]
```diff
@@ -171,8 +175,10 @@ class FrontEndOutput extends React.Component {
   }
 
   injectLocalFiles(sketchDoc) {
+    // sketchDoc.body.textContent
```
Comment (MathuraMG) [inline client/src/app/components/App/Shared/EditorComponents/CodeOutput/FrontEndOutput.jsx:178]: remove this

File: client/src/app/components/App/Shared/EditorComponents/CodeOutput/InfiniteLoopDetectableCode.js [hunk 4]
```diff
@@ -0,0 +1,44 @@
+import React from 'react';
```
Comment (MathuraMG) [inline client/src/app/components/App/Shared/EditorComponents/CodeOutput/InfiniteLoopDetectableCode.js:1]: Do we need this file?

File: client/src/components/button/button.stories.js [hunk 5]
```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import { Button } from '@storybook/react/demo';
+import InfiniteLoopDetectableCode from '../../app/components/App/Shared/EditorComponents/CodeOutput/InfiniteLoopDetectableCode.js';
```
Comment (MathuraMG) [inline client/src/components/button/button.stories.js:3]: we can remove this?

File: client/src/static/test_loop.js [hunk 6]
```diff
@@ -0,0 +1,490 @@
+/*! Loop protect | (c) 2014 JS Bin | jsbin.mit-license.org */
```
Comment (MathuraMG) [inline client/src/static/test_loop.js:1]: lets rename this

File: client/src/app/components/App/Shared/EditorComponents/CodeOutput/FrontEndOutput.jsx [hunk 7]
```diff
@@ -171,8 +175,10 @@ class FrontEndOutput extends React.Component {
...
     const scriptsToInject = [
-      '/hijackConsole.js'
+      '/hijackConsole.js',
+      '/test_loop.js'
```
Comment (MathuraMG) [inline client/src/app/components/App/Shared/EditorComponents/CodeOutput/FrontEndOutput.jsx:181]: lets rename this

**Comment by komalahuwalia** [comment]
This was already implemented as a part of other PR

**Pull request closed without merging**